hack/generate.sh

```diff
@@ -2,35 +2,35 @@
 
 # Ignore the rule that says we should always quote variables, because
 # in this script we *do* want globbing.
-# shellcheck disable=SC2086
+# shellcheck disable=SC2086,SC2292
 
 set -eux
 
-IS_CONTAINER=${IS_CONTAINER:-false}
-ARTIFACTS=${ARTIFACTS:-/tmp}
+IS_CONTAINER="${IS_CONTAINER:-false}"
+ARTIFACTS="${ARTIFACTS:-/tmp}"
 CONTAINER_RUNTIME="${CONTAINER_RUNTIME:-podman}"
 
 if [ "${IS_CONTAINER}" != "false" ]; then
-  eval "$(go env)"
-  cd "${GOPATH}"/src/github.com/metal3-io/baremetal-operator
-  export XDG_CACHE_HOME="/tmp/.cache"
+    # we need to tell git its OK to use dir owned by someone else
+    git config --global safe.directory /workdir
```

Comment:
the change looks ok
just wondering if we could define a variable instead of hardcoding /workdir since it's used in at least 2 files

Reply:
Sure, changed it in all linters as it was mentioned at least 3 times each file in Docker command alone.